src/store/rbac/actions.ts

```diff
@@ -20,6 +21,15 @@ export const fetchRbac = () => {
         dispatch(fetchRbacSuccess(res));
       })
       .catch(err => {
+        dispatch(
+          addNotification({
+            title: 'Failed to get RBAC information',
+            description:
+              'There was a problem receiving user permissions. Refreshing this page may fix it. If it does not, please contact your admin.',
```

Comment:
Can we please add this to the i18n file?

Reply:
I created the issue below, so we can have the feature in prod.

https://github.com/project-koku/koku-ui/issues/1491